qa/tasks/vstart_runner.py

```diff
@@ -583,7 +583,9 @@ def _asok_path(self):
         log.info("I think my launching pid was {0}".format(self.fuse_daemon.subproc.pid))
         return path
 
-    def mount(self, mount_path=None, mount_fs_name=None, mount_options=[]):
+    def mount(self, mount_path=None, mount_fs_name=None, mountpoint=None, mount_options=[]):
+        if mountpoint is not None:
+            self.mountpoint = mountpoint
```

Comment:
While you're in here... Should this function return an error if self.mounted is already true? It seems like you could call this once with one mountpoint and then again with another, and it would lose track of the first one. Granted I know this is mostly for testing, so I'm not clear on how robust it needs to be...

Reply:
Checked the test cases, when the mountpoint is specified with a new path, and if it umounted and reused in other followed test cases, the old mountpoint path will be lost.
We should fix them all by restoring the default mount point when umounting.